test/aleph/http_test.clj

```diff
@@ -102,14 +90,14 @@
   {:status 200
    :body (->> request :body bs/to-line-seq)})
 
-(defn hello-handler [request]
+(defn hello-handler [_request]
   {:status 200
    :body "hello"})
 
-(defn big-handler [request]
+(defn big-handler [_request]
   {:status 200
    :body (->> (s/periodically 0.1 #(byte-array 1024))
-           (s/transform (take 1e3)))})
+              (s/transform (take (long 1e3))))})
```

Comment:
Ah, right, thread macros are another instance of indentation divergence. I only changed it when I had to touch the form to fix any linter warnings, though. Note that the style I changed it to is the one recommended by [The Clojure Style Guide](https://guide.clojure.style/#threading-macros-alignment) and happens to be the default of [`clojure-mode`](https://github.com/clojure-emacs/clojure-mode). But now I know how to adjust this, so if you'd prefer, I'm happy to adapt.

Reply:
Nono, I prefer the general style guide, thanks for this.